Olha_Lesiuk/task_3.py

```diff
@@ -0,0 +1,7 @@
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Jonathan", "Artur"]
+for friend in friends[:2]:
```

Comment:
why do you use friends[:2]?

Reply:
That's because there is a condition in the task **If the friend's name is James, we don't check him because he is the best friend.**

Reply:
What happens if we shuffle our list for example to [ "James", "John", "Marta"]. When you write code it is good if your code works with different data. 

In this task what I mean by "we don't check"  is to use continue in your loop.